examples/metrics/Cargo.toml

```diff
@@ -12,13 +12,13 @@ release = false
 futures = { workspace = true }
 axum = "0.7"
 libp2p = { path = "../../libp2p", features = ["tokio", "metrics", "ping", "noise", "identify", "tcp", "yamux", "macros"] }
-opentelemetry = { version = "0.23.0", features = ["metrics"] }
-opentelemetry-otlp = { version = "0.16.0", features = ["metrics"] }
-opentelemetry_sdk = { version = "0.23.0", features = ["rt-tokio", "metrics"] }
+opentelemetry = { version = "0.25.0", features = ["metrics"] }
+opentelemetry-otlp = { version = "0.25.0", features = ["metrics"] }
+opentelemetry_sdk = { version = "0.25.0", features = ["rt-tokio", "metrics"] }
```

Comment:
any reasons not to use `0.26.0` here?

Reply:
yeah aha, `tracing-opentelemetry` depends on [`opentelemetry` `0.25`](https://crates.io/crates/tracing-opentelemetry/0.26.0/dependencies)